Cargo.toml

```diff
@@ -6,15 +6,15 @@ edition = "2021"
 [dependencies]
 rnix = "0.11.0"
 regex = "1.11.1"
-clap = { version = "4.5.20", features = ["derive"] }
-serde_json = "1.0.132"
+clap = { version = "4.5.21", features = ["derive"] }
+serde_json = "1.0.133"
 tempfile = "3.14.0"
-serde = { version = "1.0.214", features = ["derive"] }
+serde = { version = "1.0.215", features = ["derive"] }
 anyhow = "1.0"
 lazy_static = "1.5.0"
 colored = "2.1.0"
 itertools = "0.13.0"
-rowan = "0.15.16"
+rowan = "0.16.1"
```

Comment:
@philiptaron This appears to be the problem, because `rnix` uses `rowan < 0.16`, leading to two versions of rowan being used, which then fail to interact. Unfortunately just downgrading this would lead to the update script just updating it again next time, possibly caused due to https://github.com/NixOS/nixpkgs-vet/blob/700a0b2bc392160d4e488da1b3e8ddcb2ab84afa/default.nix#L111-L112 (ping @willbush :wink:)

Reply:
OK, so shift the attention to rnix and see what it takes to update to rowan 0.16. The ecosystem doesn't stop! :-)

Reply:
Oh forgot I saw your message